java.js

```diff
@@ -0,0 +1,123 @@
+/////////////main scope
+
+const input = document.getElementById("input");
+const btn = document.getElementById("btn");
+const number = document.getElementById("num");
+document.getElementById("error").style.color = "red";
+const resultsList = document.getElementById(`fullResults`);
+const checkbox = document.getElementById("check");
+const result = document.getElementById("result");
+
+  ///////////////////////////best getList function
+  
+  async function getList() {
+    const res = await fetch(`http://localhost:5050/getFibonacciResults`);
+    const data = await res.json();
+    const sortedArray = data.results.sort((a,b) =>  b.createdDate - a.createdDate);
+    appendRows(sortedArray);
+
+  }
+  
+  function appendRows(array) {
+    resultsList.innerHTML = '';
+    const list = document.createElement(`div`);
+    for(let i=0; i<array.length;i++)
+    {
+      const row= document.createElement(`div`);
+       row.innerHTML = `The Fibonacci Of: <b>${array[i].number}</b> is <b>${array[i].result}</b>. Calculated at: ${new Date(array[i].createdDate)}`;
+      list.append(row);
+      row.classList.add("fs-5", "border-bottom", "py-2", "border-secondary" );
+    }
+    resultsList.append(list);
+  
+  }
+
+// ///////////////////////////// checkbox check function
+function fibonacci(){

```

Comment:
although the function works as expected, you shoud choose better names for the variables.